builder/comp-builder.nix

```diff
@@ -179,16 +179,7 @@ let
       [ "--with-gcc=${stdenv.cc.targetPrefix}cc"
       ] ++
       # BINTOOLS
-      (if stdenv.hostPlatform.isLinux && !stdenv.hostPlatform.isAndroid # might be better check to see if cc is clang/llvm?
-        # use gold as the linker on linux to improve link times
-        then [
-          "--with-ld=${stdenv.cc.bintools.targetPrefix}ld.gold"
-          "--ghc-option=-optl-fuse-ld=gold"
-          "--ld-option=-fuse-ld=gold"
-        ] else [
-          "--with-ld=${stdenv.cc.bintools.targetPrefix}ld"
-        ]
-      ) ++ [
```

Comment:
I'm a bit worried that removing this will break cross-compilation, but can't easily test that locally.

Reply:
I checked that cross compiling still works for `pkgsCross.raspberyPi` and it all seems fine. Building a simple exe directly worked fine, and building packages using nix also seems fine. I think my worries were unfounded